code/drasil-gool/lib/Drasil/GOOL/LanguageRenderer/JuliaRenderer.hs

```diff
@@ -381,7 +381,21 @@ instance RenderValue JuliaCode where
 
 instance ValueElim JuliaCode where
   valuePrec = valPrec . unJLC
-  valueInt = valInt . unJLC
+  valueChar sc = case litVal (unJLC sc) of
```

Comment:
I think you should be able to abstract out the
```
case litVal x of
   LitChar c -> Just c
   _  -> Nothing
```
part in a separate function so that it can be re-used across the different renderers